ui/app/templates/components/mfa/mfa-login-enforcement-form.hbs

```diff
@@ -105,8 +105,7 @@
           <SearchSelect
             @placeholder="Search for an existing target"
             @options={{this.searchSelect.options}}
-            {{! workaround since there is no way provided by component to externally clear selected options }}
-            @selectedOptions={{this.searchSelect.selected}}
```

Comment:
Now that `selectedOptions` is a tracked property, we don't want to set it directly so created a new arg - ideally opening up the way to refactor for a parent to manage the selected items